### PR TITLE
Added stub for stringold module

### PR DESCRIPTION
**Pull request by ashwch**

### CODE REVIEW

File: stdlib/2/stringold.pyi [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+whitespace = ...  # type: str
+lowercase = ...  # type: str
+uppercase = ...  # type: str
+letters = ...  # type: str
+digits = ...  # type: str
+hexdigits = ...  # type: str
+octdigits = ...  # type: str
+_idmap = ...  # type: str
+_idmapL = ...  # type: Optional[List[str]]
+index_error = ...  # type: Type[ValueError]
```
Comment (JelleZijlstra) [inline stdlib/2/stringold.pyi]: You can just do `index_error = ValueError`

File: stdlib/2/stringold.pyi [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+atol_error = ...  # type: Type[ValueError]
+
+
+def lower(s: AnyStr) -> AnyStr: ...
+def upper(s: AnyStr) -> AnyStr: ...
+def swapcase(s: AnyStr) -> AnyStr: ...
+def strip(s: AnyStr) -> AnyStr: ...
+def lstrip(s: AnyStr) -> AnyStr: ...
+def rstrip(s: AnyStr) -> AnyStr: ...
+def split(s: AnyStr, sep: AnyStr = ..., maxsplit: int = ...) -> List[AnyStr]: ...
```
Comment (JelleZijlstra) [inline stdlib/2/stringold.pyi]: This (and most functions below) will probably also allow mixed str and unicode arguments.

Comment (ashwch) [inline stdlib/2/stringold.pyi]: Agreed. These definitions are mostly taken from `string.pyi`.

**Comment by ashwch** [comment]
@JelleZijlstra Even I was surprised that such module still exists in Python 2 :-)

**Pull request merged**